src/uct/sm/cma/cma_ep.c

```diff
@@ -0,0 +1,76 @@
+/**
+* Copyright (C) UT-Battelle, LLC. 2015. ALL RIGHTS RESERVED.
+* Copyright (C) Mellanox Technologies Ltd. 2001-2015.  ALL RIGHTS RESERVED.
+* $COPYRIGHT$
+* $HEADER$
+*/
+
+#define _GNU_SOURCE
```

Comment:
This is more generic issue that it is relevant for multiple others parts of the code. It should be solved out of scope of this patch